docs/design/2022-12-07-keyspace.md

```diff
@@ -0,0 +1,200 @@
+# Proposal: Keyspace
+* Authors: [ystaticy](https://github.com/ystaticy), [iosmanthus](https://github.com/iosmanthus), [AmoebaProtozoa](https://github.com/AmoebaProtozoa),[zeminzhou](https://github.com/zeminzhou)
+
+
+## Motivation
+
+We aim to store data from multiple applications in a TiKV cluster. For instance, in a multi tenant scenario, a common solution is to add a prefix to the key to differentiate the data of various applications. Therefore, we introduce a new concept referred to as `Keyspace` to describe the logical isolation among different business scenarios within a TiKV cluster.
+
+## Architecture
+
+In this section, we will outline the architecture when utilizing keyspaces.
+
+![keyspace-arch.png](imgs/keyspace-arch.png)
+
+### PD
+
+PD is responsible for managing `Keyspace` metadata. It provides HTTP and RPC interfaces to create, load, and modify `Keyspace` metadata. All other components retrieve the keyspace metadata from PD.
```

Comment:
Could you please add a raw list of the HTTP and RPC interfaces?

Reply:
Sounds good, I added links to the two interfaces below.

Reply:
So users need to call the HTTP/RPC interfaces or use pd-ctl instead of SQL?